src/site/_utils/constants.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+  paginationCount: 24,
```

Comment:
@samthor @devnook @MichaelSolati do y'all want to do SCREAMING_SNAKE_CASE for constants? I sometimes like it but I don't want to force it on folks if they find it annoying or unnecessary.

Reply:
Yes, please 👍